scrapyd/environ.py

```diff
@@ -23,16 +23,16 @@ def __init__(self, config, initenv=os.environ):
     def get_environment(self, message, slot):
         project = message['_project']
         env = self.initenv.copy()
-        env['SCRAPY_SLOT'] = str(slot)
+        env['SLOT'] = str(slot)
         env['SCRAPY_PROJECT'] = project
-        env['SCRAPY_SPIDER'] = message['_spider']
-        env['SCRAPY_JOB'] = message['_job']
+        env['SPIDER'] = message['_spider']
+        env['JOB'] = message['_job']
```

Comment:
Since these changes will make it incompatible with older versions, maybe you could also leave the old ones.
This way it will be backwards compatible (or at least I hope so)